src/minigames/shakingMinigame.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  Graphics,
+  FederatedMouseEvent,
+  Texture,
+  Assets,
+  Point,
+  Text,
+  Sprite,
+  TextStyle,
+} from "pixi.js";
+import { Minigame } from "./minigame";
+
+export class ShakingMinigame extends Minigame{
```

Comment:
Set lifetime to like 30_000, the default 10 second timeout is too short for this i think

Reply:
when playing i could shake it within 6 seconds